javascript/javascript3/week3/Getting-into-promises.js

```diff
@@ -0,0 +1,45 @@
+/*
+const li1 = document.createElement('li');
+ li1.innertext=promise.items;
+li1.appendChild(ul);*/
+const url1 = 'https://api.github.com/search/repositories?q=user:rajanibusani';
+const url2 = 'https://api.github.com/search/repositories?q=user:JyothiNandyala';
+const url3 = 'https://api.github.com/search/repositories?q=user:sofiiadidovych';
```

Comment:
similar URL is used 3 times. Can extract the long part of the URL into its own const

Reply:
Thankyou for your comments aleks.I will try to modify s soon as possible.